scripts/workers/submission_worker.py

```diff
@@ -398,9 +398,10 @@ def run_submission(challenge_id, challenge_phase, submission_id, submission, use
     with open(stdout_file, 'r') as stdout:
         stdout_content = stdout.read()
         submission.stdout_file.save('stdout.txt', ContentFile(stdout_content))
-    with open(stderr_file, 'r') as stderr:
-        stderr_content = stderr.read()
-        submission.stderr_file.save('stderr.txt', ContentFile(stderr_content))
+    if (submission_status is not Submission.FINISHED):
```

Comment:
I think the check should be 

`if (submission_status is not Submission.FAILED):`

Please correct me if I am wrong.

Reply:
@deshraj if the submission is failed then you want an Stderr file to be generated right?

Reply:
@deshraj you want an Stderr file to be generated in case of submission Failure, right?

Reply:
Yeah.

Reply:
And @deshraj 
if (submission_status is not Submission.FAILED):
this check will save an stderr file for every successful submission, right!